lib/index.js

```diff
@@ -15,6 +19,7 @@ var Appboy = module.exports = integration('Appboy')
   .option('apiKey', '')
   .option('safariWebsitePushId', '')
   .option('automaticallyDisplayMessages', true)
+  .tag('<script src="https://js.appboycdn.com/web-sdk/1.6/appboy.min.css">')
```

Comment:
@ccnixon if you include it, users can style their in-app notifications but good call in the inline comment.